scripts/generators/layer_dispatch_table_generator.py

```diff
@@ -9,6 +9,21 @@
 import os
 from generators.base_generator import BaseGenerator
 
+DEVICE_DISPATCH_TABLE_IGNORE_LIST = [
+        'vkGetDeviceProcAddr',
+]
+
+INSTANCE_DISPATCH_TABLE_IGNORE_LIST = [
+        'vkCreateInstance',
+        'vkCreateDevice',
+        'vkGetPhysicalDeviceProcAddr',
+        'vkEnumerateInstanceExtensionProperties',
+        'vkEnumerateInstanceLayerProperties',
+        'vkEnumerateInstanceVersion',
+        'vkGetInstanceProcAddr',
+]
```

Comment:
Since these globals are only used in the generate function could we define them locally instead of having them be global?